### ISSUE TITLE
Please clarify - branches  "LinkMerge" vs "LinkDaily" ?

### ISSUE THREAD

**Issue by thx1111** [realthunder/FreeCAD#879]
Hmm - "LinkDaily" has not seen an update since May, while "LinkMerge" has had updates through into August.  Is LinkDaily not really "daily" anymore?  Has the actual focus moved now to LinkMerge?  Is LinkMerge *literally* about a realthunder merge into original FreeCAD?  I'd like to stay fucused upon the "current" realthunder FreeCAD, but I'm not sure which branch that is now.
Please clarify.
Thanks

**Comment by antonok-edm** [comment]
https://github.com/realthunder/FreeCAD/issues/791#issuecomment-1556176693

> LinkStage3 is obsolete. I will maintain LinkDaily in parallel with LinkMerge. The later is for testing merge with upstream. The once found stable I'll merge it to LinkDaily.

I'm also confused by the naming, but according to this my understanding is that `LinkDaily` is the "stable" branch and `LinkMerge` is the "development" branch.
